doc/source/getting-started/administration.rst

```diff
@@ -0,0 +1,161 @@
+
+The amount of administrative reviews and approvals depends on the nature of the
+project. A project can be classified in to the following categories:
+
+.. grid:: 3
+    
+    .. grid-item-card:: :octicon:`repo` Documentation projects
+
+        Documentation projects require direct manager approval, legal
+        review, and documentation proofing. No source code other than
+        documentation files is allowed.
+
+    .. grid-item-card:: :octicon:`tools` Tools supporting products
+
+        Tool projects require direct manager and business unit's GM/VP
+        approval. No product source code is allowed.
+
+    .. grid-item-card:: :octicon:`container` Libraries supporting products
+
+        Library projects interfacing and wrapping any ANSYS, Inc. products
+        require approval of the direct manager, product line, product line, and
+        the CTO. No product source code is allowed.
+
+
+
+Legal
+^^^^^
+The legal review and approval guarantees that the whole project is subjected
+and protected by the ANSYS, Inc. legal framework.
+
+Start by completing the legal review request form:
+
+.. button-link:: https://ansys.sharepoint.com/:w:/r/sites/OpenSourceSoftwareOSSSuperintendence/_layouts/15/Doc.aspx?sourcedoc=%7B3296AD39-79EC-4F42-81C1-1DF988986800%7D&file=Open%20Source%20Policy_Request%20to%20Release%20Code_need%20GM%20sign-off_2021Sep.docx&action=default&mobileredirect=true
+    :color: black
+    :expand:
+
+    **Open Source Code Release Request Form**
+
+The following checks are required when performing the legal review of the project:
+
+.. card:: |uncheck| The project contains the right licensing
+
+    * The project contains the right license.
+    * The contribution does not contain any strong encryption.
+    * Ansys official logos and branding images are used in the project.
+    * The ANSYS, Inc. copyright appears in the right location as required by the Legal department.
+    * The copyright has the right formatting, that is ``Copyright (C) YYYY ANSYS, Inc.``.
+    * The contribution does not embody any Ansys intellectual property that is not approved for open sourcing.
+    * The contribution does not embody any invention for which Ansys has sought or received patent protection.
+    * Any third-party open sources included in the contribution have been reviewed for security vulnerabilities and have had their license files included in the repository.
+
+Open source dependencies that are not distributed as part of the project do not
+need their licenses included in the Ansys repository. Examples include
+dependent npm modules or Python packages from PyPI.
+
+
+Technical
+^^^^^^^^^
+The technical review and approval guarantees that the project follows the best
+and latest software development practices.
+
+The following checks are required when performing the technical review of the project:
+
+.. card:: |uncheck| The project contains the right metadata information
```

Comment:
I think this metadata line is a check and requires a period at the end of the sentence.